ood-portal-generator/lib/ood_portal_generator/view.rb

```diff
@@ -77,6 +82,15 @@ def initialize(opts = {})
       @register_root = opts.fetch(:register_root, nil)
     end
 
+    attr_reader :use_maintenance
+
+    # Helper method to escape IP for maintenance rewrite condition
+    def escape_ip(value)
+      # Value is already escaped
+      return value if value.split(%r{\\.}, 4).size == 4
+      value.split('.', 4).join('\.')
```

Comment:
like in the other place, i think that when you have a guard clause like this and only one or two lines following it is easier to read the code if changed to using an if/else block:

```ruby
def escape_ip(value)
  if value.split(%r{\\.}, 4).size == 4
    value
  else
    value.split('.', 4).join('\.')
  end
end
```

Reply:
Fixed